packages/synapse-react-client/src/components/DownloadCart/DownloadListTable.tsx

```diff
@@ -257,6 +257,7 @@ export default function DownloadListTable(props: DownloadListTableProps) {
             <tbody>
               {allRows.map((item: DownloadListItemResult) => {
                 if (item) {
+                  item.versionNumber = null
```

Comment:
Did you intend to check this in (always setting the item version number to null)?

Reply:
Sorry! This is a line that I put to make sure the solution work. Let me remove it